src/app/tests/suites/OTA_SuccessfulTransfer.yaml

```diff
@@ -0,0 +1,177 @@
+# Copyright (c) 2022 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Test OTA Software Update Successful Transfer
+
+config:
+    endpoint: 0
+    requestorNodeId:
+        type: NODE_ID
+        defaultValue: 0x12344321
+    providerNodeId:
+        type: NODE_ID
+        defaultValue: 0xC0FFEE
+    providerPayload:
+        type: CHAR_STRING
+        defaultValue: "MT:-24J0IX4122-.548G00" # This value needs to be generated
+    providerDiscriminator:
+        type: INT16U
+        defaultValue: 50
+    providerPort:
+        type: INT16U
+        defaultValue: 5560
+    providerKvs:
+        type: CHAR_STRING
+        defaultValue: "/tmp/chip_kvs_provider"
+    otaImageFilePath:
+        type: CHAR_STRING
+        defaultValue: "/tmp/otaImage"
+    rawImageFilePath:
+        type: CHAR_STRING
+        defaultValue: "/tmp/rawImage"
+    rawImageContent:
+        type: CHAR_STRING
+        defaultValue: "Have a hootenanny!"
+    downloadImageFilePath:
+        type: CHAR_STRING
+        defaultValue: "/tmp/downloadedImage"
```

Comment:
Why not have scripts generate these filenames using these as prefix: would allow better collision avoidance with multiple runs on same machine (i.e. add a run-specific suffix to these paths)

Reply:
Can do as follow-up